deploy/cluster-operator/templates/admission-registration.yaml

```diff
@@ -0,0 +1,20 @@
+# Deprecated in v1.16 in favor of admissionregistration.k8s.io/v1
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: "cluster-validator"
+webhooks:
+- name: "cluster-validator.operator.svc"
+  rules:
+  - apiGroups:   ["cluster-operator.infobloxopen.github.com"]
+    apiVersions: ["v1alpha1"]
+    operations:  ["UPDATE", "CREATE"]
+    resources:   ["clusters"]
+    scope:       "Namespaced"
+  clientConfig:
+    service:
+      namespace: "operator"
+      name: "cluster-validator"
+    caBundle: ${CA_PEM_B64}
```

Comment:
This needs to reference a real ca bundle. I recommend using cert-manager to populate it https://cert-manager.io/docs/concepts/ca-injector/